index.js

```diff
@@ -1,8 +1,6 @@
 /* jshint node: true */
```

Comment:
Oh that file, not the other one.  Mmmmh, not sure. I'll leave it here and ask Rob.